src/plugins/controls/common/options_list/mocks.tsx

```diff
@@ -20,7 +23,7 @@ const mockOptionsListComponentState = {
   availableOptions: ['woof', 'bark', 'meow', 'quack', 'moo'],
   invalidSelections: [],
   validSelections: [],
-  searchString: { value: '', valid: true },
+  ...getDefaultComponentState(),
```

Comment:
Nice addition here to make it more modular! 

> Super tiny nit that doesn't make a difference now but _might_ later

Because the default spread happens at the end, if we ever introduced something to the default state, like `availableOptions: []` for instance, it would **overwrite** the additional state in this test which could cause test failures. I would move this to the top so anything declared in these mocks overrides the default.

Reply:
Oh! Yeah, that's a good point. Fixed 👍